product_pricelist_fixed_currency_rate/views/pricelist_views.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_pricelist_item_form_view" model="ir.ui.view">
+        <field
+            name="name"
+        >product.pricelist.item.form - product_pricelist_fixed_currency_rate</field>
+        <field name="model">product.pricelist.item</field>
+        <field name="inherit_id" ref="product.product_pricelist_item_form_view" />
+        <field name="arch" type="xml">
+            <field name="base_pricelist_id" position="after">
+                <field name="is_fixed_currency_rate_applicable" invisible="1" />
+                <field name="do_inverse_currency_rate" invisible="1" />
+                <field
+                    name="currency_rate_tooltip"
+                    nolabel="1"
+                    attrs="{
+                    'invisible': [('is_fixed_currency_rate_applicable', '=', False)]}"
```

Comment:
The migration tasks have not been done. Please follow the tasks as mentioned in: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#tasks-to-do-in-the-migration